pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/kitrequest/KitRequestDao.java

```diff
@@ -192,6 +193,6 @@ public KitRequestShipping getKitRequestForCollaboratorSampleId(String collaborat
             throw new DsmInternalError("Error getting KitRequestShipping by collaboratorSampleId "
                     + collaboratorSampleId, results.resultException);
         }
-        return (KitRequestShipping) results.resultValue;
+        return kitRequestShippingList;
```

Comment:
I realized that collaborator sample id is not a unique column by constraint, so here I have changed the code to get all of the results with this collaborator sample id. In the methods that call this method I throw an exception if the list that is returned have more than one value since that is an odd situation